View Controllers/LoginViewController.m

```diff
@@ -81,17 +81,11 @@ - (IBAction)didTapLogin:(id)sender {
 
 - (void)didReceiveMemoryWarning {
     [super didReceiveMemoryWarning];
-    // Dispose of any resources that can be recreated.
 }
 
-/*
```

Comment:
Will the app still compile if we delete  - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 as well?